router/network.go

```diff
@@ -157,6 +175,9 @@ func (network *netHandle) SendPost(ctx context.Context, structData integrations.
 		}
 
 		req.Header.Add("User-Agent", "RudderLabs")
+		if bodyFormat == "GZIP" {
+			req.Header.Add("Content-Encoding", "gzip")
+		}
```

Comment:
We are already getting the headers with the transform event. Can't we skip this step?

Reply:
Yes, removed the redundant header.